### ISSUE TITLE
Crash FATAL EXCEPTION: OkHttp Dispatcher

### ISSUE THREAD

**Issue by RazibKani** [Instabug/Instabug-Android#387]
Hi, I got crash when try to implement Instabug APM (10.6.0).
It happen when I implement this:
```
apply plugin: 'instabug-apm'

implementation "com.instabug.library:instabug-apm-okhttp-interceptor:10.6.0"
```

This is from my stack trace:
```
2021-06-11 17:12:56.030 28713-28989/com.happyfresh.fulfillmentv2.staging E/AndroidRuntime: FATAL EXCEPTION: OkHttp Dispatcher
    Process: com.happyfresh.fulfillmentv2.staging, PID: 28713
    java.lang.IllegalStateException: closed
        at okio.RealBufferedSource.request(RealBufferedSource.kt:203)
        at com.instabug.library.okhttplogger.InstabugOkhttpInterceptor.populateNetworkResponse(InstabugOkhttpInterceptor.java:265)
        at com.instabug.library.okhttplogger.InstabugOkhttpInterceptor.intercept(InstabugOkhttpInterceptor.java:97)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
        at com.chuckerteam.chucker.api.ChuckerInterceptor.intercept(ChuckerInterceptor.kt:85)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
        at com.happyfresh.snowflakes.hoverfly.interceptors.GlobalErrorInterceptor.intercept(GlobalErrorInterceptor.kt:19)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
        at com.happyfresh.snowflakes.hoverfly.interceptors.ClientInterceptor.intercept(ClientInterceptor.kt:61)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
        at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:201)
        at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:517)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:919)
```

Thank you.

### Steps to Reproduce the Problem
First run my app and it crash

### Expected Behavior
No crash occur

### Actual Behavior
Crash occur

### SDK Version
10.6.0

### Android Version
Android 10

### Device Model
Poco X3 NFC

**Comment by MShannan** [comment]
Thanks for reaching out @RazibKani, we'll look into it.

**Comment by RazibKani** [comment]
Hi @MShannan, 
Do you have any update regarding this issue?

Thanks.

**Comment by MShannan** [comment]
Hi @RazibKani 
we can’t replicate it w it would be helpful if you could replicate the issue on our sample app.

**Comment by MShannan** [comment]
Hi @RazibKani 
We created a SNAPSHOT that should fix the issue, Can you please try it and tell us if it fixed the issue for you?

`implementation 'com.instabug.library:instabug:10.7.0.3298-SNAPSHOT'`
you will need to add these lines to your app build.gradle before your dependencies

```
repositories {
    ....
    maven {
        url "https://oss.sonatype.org/content/repositories/snapshots"
    }
}
```
This SNAPSHOT is completely safe to use and has no debug code at all.

Thank You!

**Comment by mzelzoghbi** [comment]
Hi @RazibKani 
This issue has been fixed in 10.7.1, please try it out and let us know if you encountered any further issues. 
Thanks
